airbyte-config/config-models/src/main/resources/types/SyncStats.yaml

```diff
@@ -7,13 +7,17 @@ type: object
 required:
   - recordsEmitted
   - bytesEmitted
-additionalProperties: false
+additionalProperties: true
 properties:
   recordsEmitted:
     type: integer
   bytesEmitted:
     type: integer
-  stateMessagesEmitted: # TODO make required once per-stream state messages are supported in V2
+  sourceStateMessagesEmitted: # TODO make required once per-stream state messages are supported in V2
```

Comment:
OPT: I would remove this todo. I don't think we benefit from marking this attribute as required so I would not add this constraint.